### PR TITLE
[TASK] Move the Composer script descriptions to the `composer.json`

### PR DESCRIPTION
**Pull request by oliverklee**
Composer has a built-in functionality for describing Composer scripts,
and we should use it for documentating our custom scripts.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -145,6 +145,28 @@
 			"rm phpcs.xml"
 		]
 	},
+	"scripts-descriptions": {
+		"ci": "Runs all dynamic and static code checks.",
+		"ci:composer:normalize": "Checks the composer.json.",
+		"ci:dynamic": "Runs alle PHPUnit tests (unit and functional).",
```
Comment (lukaszuznanski) [inline composer.json:151]: "all"

File: composer.json [hunk 2]
```diff
@@ -145,6 +145,28 @@
...
+		"ci:json:lint": "Lints the JSON files.",
+		"ci:php": "Runs all static checks for the PHP files.",
+		"ci:php:copypaste": "Checks for copy'n'pasted PHP code.",
+		"ci:php:cs-fixer": "Checks the code style with the PHP Coding Standards Fixer (PHP-CS-Fixer).",
+		"ci:php:lint": "Lints the PHP files for syntax errors.",
+		"ci:php:sniff": "Checks the code style with PHP_CodeSniffer (PHPCS).",
+		"ci:php:stan": "Checks the PHP types using PHPStan.",
+		"ci:static": "Runs all static code checks (syntax, style, types).",
+		"ci:tests": "Runs alle PHPUnit tests (unit and functional).",
```
Comment (lukaszuznanski) [inline composer.json:160]: "all"

File: README.md [hunk 3]
```diff
@@ -127,27 +127,6 @@ before you can use the Composer scripts.
 You can run `composer` (or `ddev composer`) to display a list of all available
 Composer commands and scripts.
 
-| Script name            | Action                        | Tool                                                                            |
```
Comment (lukaszuznanski) [inline README.md:130]: should we also mention that all scripts descriptions are in composer.json in script-description section?

**Review by lukaszuznanski — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (oliverklee) [reply composer.json]: Done (in two places).

File: README.md (hunk 3 above)
Comment (oliverklee) [reply README.md:130]: Done.

**Review by lukaszuznanski — APPROVED** [review]

**Pull request merged**